pubtools/_quay/utils/tracing.py

```diff
@@ -0,0 +1,121 @@
+"""Instrument Pub task functions.
```

Comment:
As this is most probably going to be used by different projects as well, it would make sense to move it to pubtools project

Reply:
Make sense, let me move it.

Reply:
Move it to pubtools: https://github.com/release-engineering/pubtools/pull/149/